src/screens/profile-screen/CollectionList.tsx

```diff
@@ -0,0 +1,49 @@
+import { Collection } from 'audius-client/src/common/models/Collection'
+import { ProfileUser } from 'audius-client/src/common/store/pages/profile/types'
+import { FlatList, Text } from 'react-native'
+
+import { CollectionCard } from '../explore-screen/components/CollectionCard'
+
+import { EmptyTab } from './EmptyTab'
+
+type CollectionListProps = {
+  collection: Collection[]
+  profile: ProfileUser
+  emptyTabText: string
+}
+
+export const CollectionList = ({
+  collection,
+  profile,
+  emptyTabText
+}: CollectionListProps) => {
+  return (
+    <FlatList
+      data={collection}
+      renderItem={({ item, index }) => {
+        const isInLeftColumn = !(index % 2)
```

Comment:
Could the 44% maybe be achieved by adding an extra padding/margin to the hanging item? That way it could be a constant 16px for example instead of a percentage?

Reply:
yeah can definitely try this, i was getting strange shadow behavior, maybe a quick huddle?